client/src/components/ChatBox/ContextModal.tsx

```diff
@@ -16,7 +16,7 @@ const ContextModal = ({
     <MyModal
       isOpen={true}
       onClose={onClose}
-      title={`完整对话记录(${context.length}条)`}
+      title={t(`full conversation log(${context.length}strip)`)}
```

Comment:
If it is a translation with variables, you need to write:
t('chat.Full Context n strip', {length: context.length})

en.json:
```json
"chat":{
    "Full Context n strip": "Full context ({{length}}) strip"
}
```

zh.json
```json
"chat":{
    "Full Context n strip": "完整对话记录 ({{length}}) 条"
}
```